include/swift/Option/FrontendOptions.td

```diff
@@ -1023,6 +1023,11 @@ def cxx_interop_use_opaque_pointer_for_moveonly :
   HelpText<"Testing flag that will be eliminated soon. Do not use.">,
   Flags<[FrontendOption, HelpHidden]>;
 
+def cxx_modules_require_objc :
+    Separate<["-"], "cxx-modules-require-objc">, MetaVarName<"<name>">,
```

Comment:
Do you think it would be more clear if we say `-clang-modules-...` rather than `-cxx-modules-...`, since Clang uses the term `cxx-modules` for C++20 modules?

Reply:
Instead of adding an entirely new option, you could use the existing `-enable-experimental-feature` syntax so that existing compilers accept and ignore it, e.g.,

```
-enable-experimental-feature RequiresObjC=ModuleA,ModuleB
```